uktrade/data-hub-frontend
# Remove the current company from the D&B Hierarchy page

## Description of change

Remove the current company from the D&B Hierarchy page to make the number of related companies consistent with the business details page.

## Test instructions

Go to the Global Ultimate page and check if this company is not listed with the rest of D&B related companies.
 
## Screenshots
### Before

![image](https://user-images.githubusercontent.com/4199239/70902471-c4a74800-1ff4-11ea-82d2-7046f693299d.png)


### After 

![image](https://user-images.githubusercontent.com/4199239/70902452-b78a5900-1ff4-11ea-8297-f9a0fabb991f.png)


## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied. 
https://github.com/uktrade/data-hub-frontend/blob/develop/docs/Code%20review%20guidelines.md"
